spec.bs

```diff
@@ -670,15 +800,20 @@ as appropriate:
     1. Let |globalScopes| be |this|'s [=Worklet/global scopes=].
     1. [=Assert=]: |globalScopes|' [=list/size=] equals 1.
     1. Set |globalScopes|[0]'s {{SharedStorageWorkletGlobalScope/
-        privateAggregation}}'s [=PrivateAggregation/get batching scope steps=]
-        to the following algorithm:
-        1. Return the [=batching scope=] that is scheduled to be passed to
-            [=process contributions for a batching scope=] when the currently
-            executing call returns.
-
-            Note: Multiple operation invocations can be in-progress at the same
-            time, each with a different batching scope. However, only one can be
-            currently executing.
+        privateAggregation}}'s [=PrivateAggregation/scoping details=] a new
+            [=scoping details=] with the items:
+        : [=scoping details/get batching scope steps=]
+        :: An algorithm that returns the [=batching scope=] that is scheduled to
+            be passed to [=process contributions for a batching scope=] when the
+            call currently executing in |scope| returns.
+        : [=scoping details/get debug scope steps=]
+        :: An algorithm that returns the [=debug scope=] that is scheduled to be
+            passed to [=mark a debug scope complete=] when the call currently
+            executing in |scope| returns.
+
+        Note: Multiple operation invocations can be in-progress at the same
```

Comment:
Is this argument the same for debug scope?

Reply:
Yep, edited to reflect that